lib/src/booleans/boolean_contains.dart

```diff
@@ -1,75 +1,75 @@
 import 'package:turf/turf.dart';
 
-import '../invariant.dart';
 import 'boolean_point_in_polygon.dart';
 import 'boolean_point_on_line.dart';
 
-/**
- * Boolean-contains returns True if the second geometry is completely contained by the first geometry.
- * The interiors of both geometries must intersect and, the interior and boundary of the secondary (geometry b)
- * must not intersect the exterior of the primary (geometry a).
- * Boolean-contains returns the exact opposite result of the `@turf/boolean-within`.
- *
- * @name booleanContains
- * @param {Geometry|Feature<any>} feature1 GeoJSON Feature or Geometry
- * @param {Geometry|Feature<any>} feature2 GeoJSON Feature or Geometry
- * @returns {boolean} true/false
- * @example
- * var line = turf.lineString([[1, 1], [1, 2], [1, 3], [1, 4]]);
- * var point = turf.point([1, 2]);
- *
- * turf.booleanContains(line, point);
- * //=true
- */
+/// Boolean-contains returns True if the second geometry is completely contained by the first geometry.
+/// The interiors of both geometries must intersect and, the interior and boundary of the secondary (geometry b)
+/// must not intersect the exterior of the primary (geometry a).
+/// Boolean-contains returns the exact opposite result of the `@turf/boolean-within`.
+/// @name booleanContains
+/// @param {Geometry|Feature<any>} feature1 GeoJSON Feature or Geometry
+/// @param {Geometry|Feature<any>} feature2 GeoJSON Feature or Geometry
+/// @returns {boolean} true/false
+/// example:
+/// ```dart
+/// var line = turf.lineString([[1, 1], [1, 2], [1, 3], [1, 4]]);
+/// var point = turf.point([1, 2]);
+/// turf.booleanContains(line, point);
+/// //=true
+/// ```
 booleanContains(GeoJSONObject feature1, GeoJSONObject feature2) {
-  var geom1 = getGeom(feature1);
-  var geom2 = getGeom(feature2);
-  var type1 = geom1.runtimeType;
-  var type2 = geom2.runtimeType;
-  var coords1 = geom1.coordinates;
-  var coords2 = geom2.coordinates;
+  var geom1 = feature1 is Feature ? feature1.geometry : feature1;
+  var geom2 = feature2 is Feature ? feature2.geometry : feature2;
+  var type1 = geom1!.type;
+  var type2 = geom2!.type;
+  var coords1 = (geom1 as GeometryType).coordinates;
+  var coords2 = (geom2 as GeometryType).coordinates;
```

Comment:
@lukas-h, 
this would be a place where implementing [this](https://github.com/dartclub/turf_dart/pull/91/files#r893679464) would result to.  
I will resolve the other conversation when you review this.

Reply:
I don't really know what this is referring to?  `getGeom` by chance?

Reply:
Right. We wanted to get rid of getGeom().